src/test/java/edu/cornell/kfs/concur/batch/fixture/ConcurRequestExtractRequestDetailFileLineFixture.java

```diff
@@ -11,23 +11,26 @@
 import edu.cornell.kfs.concur.batch.businessobject.ConcurRequestExtractRequestEntryDetailFileLine;
 
 public enum ConcurRequestExtractRequestDetailFileLineFixture {
+
+    REQUEST_DETAIL_GOOD_LINE_1((new Date(Calendar.getInstance().getTimeInMillis())), new String("34YA"), new KualiDecimal(10.00), new String("CORNELL"), 1),
```

Comment:
I have a couple suggestions on how you could make these fixtures easier to read and maintain:

[1] You could add a shortcut constructor for defaulting to the current time if desired. Also, if your tests are time-sensitive, it may be better to pick a static date instead of always using the current one.

[2] The "new String()" constructors are redundant; it would be cleaner to use constants or literals directly. If you absolutely need to test things using different String instances, then I would recommend doing the "new String()" calls within the body of the constructor.

[3] To simplify the setup of amounts, you could modify the constructor to accept a primitive double, and then convert it to a KualiDecimal within the body of the constructor.